optimize_sidebars.js

```diff
@@ -0,0 +1,1053 @@
+function docsComponentsLink(label, href) {
+  return {
+    type: "link",
+    label: label,
+    href: `/docs/next/${href}`,
+  };
+}
```

Comment:
Long term vision: 

I could have generated this file without this helper, but it would have resulted in `/docs/next/` littered throughout it. This seems like it'd be not fun when we generate a new version of optimize -- we'd have to do a find/replace and include all those changes in the version PR. 

Extracting the prefix of the paths to this one spot means we can just replace it in this one spot when we cut a new version. 

It might be YAGNI, but it made sense to me.